wd_demo.au3

```diff
@@ -1263,3 +1264,47 @@ Func SetupOpera($bHeadless)
 	Local $sCapabilities = _WD_CapabilitiesGet()
 	Return $sCapabilities
 EndFunc   ;==>SetupOpera
+
+Func SetupEdgeIEMode() ; this is for MS Edge IE Mode
+	#REMARK In order to properly use MS Edge In IE Mode you need To Do some settings - Set the "Internet, Local intranet, Trusted sites, Restricted sites" to protected mode, enabling or disabling it is not a matter
+	; https://www.edureka.co/community/1690/problem-using-internetexplorerdriver-selenium-webdriver
+	Local $sTimeStamp = @YEAR & '-' & @MON & '-' & @MDAY & '_' & @HOUR & @MIN & @SEC
+	; https://www.selenium.dev/documentation/ie_driver_server/#required-configuration
+	_WD_Option('Driver', 'IEDriverServer.exe') ;
+	_WD_Option('Port', 5555)
+	_WD_Option('DriverParams', '--verbose --log-path="' & @ScriptDir & '\' & $sTimeStamp & ' EdgeIEMode.log" -log-level=INFO' & " --port=" & $_WD_PORT & " -host=127.0.0.1" )
+#cs
+	Local $sCapabilities = '{"capabilities": {"alwaysMatch": { "se:ieOptions" : { "ie.edgepath":"C:\\Program Files (x86)\\Microsoft\\Edge\\Application\\msedge.exe", "ie.edgechromium":true, "ignoreProtectedModeSettings":true,"excludeSwitches": ["enable-automation"]}}}}'
+	Local $sCapabilities = _
+			'{' & @CRLF & _
+			'    "capabilities": {' & @CRLF & _
+			'        "alwaysMatch": {' & @CRLF & _
+			'            "se:ieOptions": {' & @CRLF & _
+			'                "ie.edgepath": "C:\\Program Files (x86)\\Microsoft\\Edge\\Application\\msedge.exe",' & @CRLF & _
+			'                "ie.edgechromium":true,' & @CRLF & _
+			'                "ignoreProtectedModeSettings":true,' & @CRLF & _
+			'                "excludeSwitches": ["enable-automation"]' & @CRLF & _
+			'            }' & @CRLF & _
+			'        }' & @CRLF & _
+			'    }' & @CRLF & _
+			'}' & @CRLF & _
+			''
+	ConsoleWrite("! $sCapabilities (by hand) = " & @CRLF & $sCapabilities & @CRLF)
+#ce
+
+	_WD_CapabilitiesDefine($_WD_KEYS__SPECIFICVENDOR_PRIMITIVE, "ie.edgepath")
+	_WD_CapabilitiesDefine($_WD_KEYS__SPECIFICVENDOR_PRIMITIVE, "ie.edgechromium")
+	_WD_CapabilitiesDefine($_WD_KEYS__SPECIFICVENDOR_PRIMITIVE, "ignoreProtectedModeSettings")
+
+	_WD_CapabilitiesStartup()
+	_WD_CapabilitiesAdd('alwaysMatch', 'msedgeie')
+	_WD_CapabilitiesAdd('w3c', True)
+	_WD_CapabilitiesAdd("ie.edgepath", "C:\Program Files (x86)\Microsoft\Edge\Application\msedge.exe")
```

Comment:
Use `_WD_GetBrowserPath` instead of hardcoding the path.

Reply:
Take a look here: 
[_WD_GetBrowserPath("msedge")](https://github.com/Danp2/au3WebDriver/pull/454/commits/6b84292ac8d6b89c02b45d98084771afa090a01f)